### DISCUSSION TITLE
qBitrr Issues

### DISCUSSION THREAD

**Discussion by hggh12345** [Feramance/qBitrr#63] [Q&A] [answered]
Hey, I have just got qBitrr up and running after many a problem. I actually only managed to get it working at all after I used the config file posted to assist tyrannical69 in another thread. I am getting two issues however, one, new released torrents from radarr seem to be getting removed due to the following: "qBitrr.Radarr-1080: Deleting Stale torrent: Unavailable | [Progress: 4.6%][Added On: 2024-02-22 18:23:49][Availability: 24.7%][Time Left: 2:26:59][Last active: 2024-02-22 18:27:00] | [TorrentState.DOWNLOADING] | Wonka 2023 2160p UHD BluRay REMUX DV HDR HEVC Atmos-TRiToN" Do I need to reconfigure something in the config file to avoid this? 

The second issue is Sonarr is doing a loop of adding a file, removing it within a couple of minutes and blacklisting it, then sonarr is grabbing the same upload again over and over again. Once again, is there a solution here?

I am running this all through Unraid through the Docker template.

**Comment by hggh12345** [comment]
I manually grabbed the same Wonka torrent and it was removed again, however with the following:
"qBitrr.Radarr-1080: Deleting Stale torrent: Unavailable | [Progress: 32.62%][Added On: 2024-02-22 18:40:01][Availability: 32.9%][Time Left: 4:58:59][Last active: 2024-02-22 19:03:03] | [TorrentState.DOWNLOADING] | Wonka 2023 2160p UHD BluRay REMUX DV HDR HEVC Atmos-TRiToN"

**Comment by Feramance** [comment]
I'll take a look at this tomorrow. Please attach the torrent file or magnet link so I can test it

**Reply by hggh12345** [reply]
magnet:?xt=urn:btih:0280EC7D45CC20722259A47B80F67A3EE38BD541&dn=Wonka%202023%202160p%20UHD%20BluRay%20REMUX%20DV%20HDR%20HEVC%20Atmos-TRiToN&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337&tr=udp%3A%2F%2Ftracker.openbittorrent.com%3A6969%2Fannounce&tr=udp%3A%2F%2Fopen.stealth.si%3A80%2Fannounce&tr=udp%3A%2F%2Ftracker.torrent.eu.org%3A451%2Fannounce&tr=udp%3A%2F%2Ftracker.bittor.pw%3A1337%2Fannounce&tr=udp%3A%2F%2Fpublic.popcorn-tracker.org%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.dler.org%3A6969%2Fannounce&tr=udp%3A%2F%2Fexodus.desync.com%3A6969&tr=udp%3A%2F%2Fopentracker.i2p.rocks%3A6969%2Fannounce

**Comment by Feramance** [comment]
Also, if sonarr is grabbing the same torrent that appears to be blacklisted, that's a sonarr issue or rather, an indexer issue. You have multiple indexers responding with the same torrent and Sonarr needs to block them all. qBitrr doesn't grab any torrents, or search, it just tells sonarr or radarr to do so

**Comment by Feramance** [comment]
Just letting you know I haven't forgotten about this, for the time being manually tag such torrents with the `qbitrr-ignored` tag that should be present on your qbittorrent instance after running qBitrr once

**Comment by Feramance** [comment]
Currently debugging to resolve this, noticed it with some other torrents

**Reply by hggh12345** [reply]
Perfect thankyou

**Comment by Feramance** [comment, marked as answer]
Give the torrent another try, it seems to be working

**Reply by Feramance** [reply]
@hggh12345 any updates?
